nixos/modules/services/games/archisteamfarm.nix

```diff
@@ -163,29 +166,29 @@ in
   config = mkIf cfg.enable {
 
     users = {
-      users.asf = {
+      users.archisteamfarm = {
         home = cfg.dataDir;
         isSystemUser = true;
-        group = "asf";
+        group = "archisteamfarm";
         description = "Archis-Steam-Farm service user";
       };
-      groups.asf = { };
+      groups.archisteamfarm = { };
     };
 
     systemd.services = {
-      asf = {
+      archisteamfarm = {
         description = "Archis-Steam-Farm Service";
         after = [ "network.target" ];
         wantedBy = [ "multi-user.target" ];
 
         serviceConfig = mkMerge [
-          (mkIf (cfg.dataDir == "/var/lib/asf") {
-            StateDirectory = "asf";
+          (mkIf (cfg.dataDir == stateDirectory) {
+            StateDirectory = "archisteamfarm";
```

Comment:
```suggestion
            StateDirectory = if lib.versionAtLeast config.system.stateVersion "24.05" then "archisteamfarm" else "asf";
```

or

```suggestion
            StateDirectory = lib.last (lib.splitString "/" stateDirectory);
```

Reply:
Tested on my deployment and it was a noop.

Reply:
That's odd. Is your state version is 24.05?